### PR TITLE
Update translation.json

### PR DESCRIPTION
**Pull request by fomod-in**
major update italian translation

### CODE REVIEW

File: packages/apps/public/locales/it/translation.json [hunk 1]
```diff
@@ -200,11 +251,15 @@
...
   "Do not include a tip for the block author": "Non includere una mancia per il minatore del blocco",
+  "Do not transfer any funds directly to a specific account that is associated with a loan or a team. Use the \"Contribute\" action to record the contribution on-chain using the crowdloan runtime module. When the fund is dissolved, after either the parachain lease expires or the loan ending without winning, the full value will be returned to your account by the runtime. Funds sent directly to an account, without using the crowdloan functionality, may not be returned by the receiving account.": "Non trasferire fondi direttamente ad un account specifico associato ad un asta o ad un team. Usa il modulo \"Contribute\" per registrare un contributo on-chain usando la funzione di Crowdloan. Quando il finanziamento è estinto, dopo la scadenza del tempo di leasing della parachain o un asta senza aggiudicazione, il saldo ti sarà restituito automaticamente. I fondi inviati direttamente senza l'utilizzo della funzione di Crowdloan non saranno restituiti"
```
Comment (jacogr) [inline packages/apps/public/locales/it/translation.json:254]:
```suggestion
  "Do not transfer any funds directly to a specific account that is associated with a loan or a team. Use the \"Contribute\" action to record the contribution on-chain using the crowdloan runtime module. When the fund is dissolved, after either the parachain lease expires or the loan ending without winning, the full value will be returned to your account by the runtime. Funds sent directly to an account, without using the crowdloan functionality, may not be returned by the receiving account.": "Non trasferire fondi direttamente ad un account specifico associato ad un asta o ad un team. Usa il modulo \"Contribute\" per registrare un contributo on-chain usando la funzione di Crowdloan. Quando il finanziamento è estinto, dopo la scadenza del tempo di leasing della parachain o un asta senza aggiudicazione, il saldo ti sarà restituito automaticamente. I fondi inviati direttamente senza l'utilizzo della funzione di Crowdloan non saranno restituiti",
```

Missing comma that makes the build fail.

**Review by fomod-in — COMMENTED** [review]
missed comma fixed

**Review by fomod-in — COMMENTED** [review]
missed comma fixed

**Review by jacogr — APPROVED** [review]
Thank you.

**Pull request merged**